### PR TITLE
[Bug] Fixing not correct path in Readme

### PR DESCRIPTION
**Pull request by Tokesh**
### Description
First of all, './dockerfiles/integtest-runner.al2.dockerfile' this file does not exist. So we should replace it by '<Docker file path' anyway

In the readme, we say that we are launching on one of the platforms('either x64 or arm64'), that is, the single arch. So here probably its more suitable to use 'bash build-image-single-arch.sh' instead of 'docker-build'?
I think current Readme not newbie-friendly, so i tried to change it. I hope i understand all differences well (between single arch and multi-arch and etc). 
I'm ready to hear all your feedback and change my mind in case of mistakes! I'm still a beginner, but I'm trying to improve the problems that I have myself

### Issues Resolved
[#3458]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ We build, assemble, and test our artifacts on docker containers. All of our pipe
 To build the docker image for either x64 or arm64, run the below command on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
 
 ```bash
-docker build -f ./dockerfiles/integtest-runner.al2.dockerfile . -t <Docker Hub RepoName>/<Docker Image Name>:<Tag Name>
+bash build-image-single-arch.sh -r <Repo name> -v <Tag name> -f <Docker File Path>
```
Comment (peterzhuamazon) [inline docker/README.md:14]: No need to add `bash` specifically for the comment, just use `./`.

**Review by peterzhuamazon — COMMENTED** [review]
Thanks @Tokesh.

Add a comment to this one.

**Comment by peterzhuamazon** [comment]
As explained in this post it is meant to use the build docker image scripts to build docker.
We havent update the readme for a while: https://forum.opensearch.org/t/opensearch-build-troubles/14218/3?u=zhujiaxi

Thanks.

**Comment by Tokesh** [comment]
> As explained in this post it is meant to use the build docker image scripts to build docker. We havent update the readme for a while: https://forum.opensearch.org/t/opensearch-build-troubles/14218/3?u=zhujiaxi
> 
> Thanks.
Ohh, I think I'm beginning to understand.
If we want to use these dockerfiles, is it enough for us to pull them into the docker(like 'docker pull opensearchstaging/ci-runner') and start from it, than to run the docker build of each dockerfile every time?
So we need to rewrite the entire readme of this directory?

**Comment by peterzhuamazon** [comment]
> > As explained in this post it is meant to use the build docker image scripts to build docker. We havent update the readme for a while: https://forum.opensearch.org/t/opensearch-build-troubles/14218/3?u=zhujiaxi
> > Thanks.
> 
> Ohh, I think I'm beginning to understand. If we want to use these dockerfiles, is it enough for us to pull them into the docker(like 'docker pull opensearchstaging/ci-runner') and start from it, than to run the docker build of each dockerfile every time? So we need to rewrite the entire readme of this directory?

Hi,
The change you made is related to how to build the image, so your PR is fine just need to address the comment I added.
Then we can merge this PR.

As to how to use the image, those images are meant to be used with automation tools. You can use by yourself but we dont have guide on those, as everything is already taking care of by automation on Jenkins.

See directory to find all the ways we are using the images.
Thanks.

**Comment by Tokesh** [comment]
@peterzhuamazon Hello! Thank you for a such detailed answer. I got it!

**Comment by aabeshov** [comment]
Hello @peterzhuamazon!
Because we assumed that intro to build multi-arch image is also out of date.
So I rewrote instructions of building multi-arch docker images.

**Comment by peterzhuamazon** [comment]
Hi @Tokesh @aabeshov I am ok with your change, but after reviewing a bit I think it is logical to move this `README.md` to the `docker/ci` directory instead of keeping it in `docker` directory, as the `docker/release` directory already have a corresponding README file.

This `docker/README.md` is mostly related to `docker/ci` so no issues moving there in my opinion.

I will move it and merge this PR, thanks.

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**